src/js/smooch.jsx

```diff
@@ -58,7 +58,7 @@ function renderLink() {
 function getDeviceId() {
     const SK_STORAGE = 'sk_deviceid';
     const deviceId = storage.getItem(SK_STORAGE) ||
-        uuid.v4().replace(/-/g, '');
+    uuid.v4().replace(/-/g, '');
```

Comment:
Atom does the same :confused: 
![atom-esformatter](https://cloud.githubusercontent.com/assets/1034297/13853590/c0d75c2c-ec3d-11e5-98f1-4e965e0504f2.gif)